lib/util/defaultBlocksToLiveRoutesDB.ts

```diff
@@ -0,0 +1,43 @@
+import { ChainId } from '@uniswap/sdk-core'
+
+// This is not being used in production today anyway, due to below filterExpiredCachedRoutes method not really filtering on the blocks-to-live
+// heuristic is within 30 seconds we find a route.
+// For the Ratio we are approximating Phi (Golden Ratio) by creating a fraction with 2 consecutive Fibonacci numbers
+
+// changing to this way with ChainId enum as key indexing, so that we wont forgot to add new chain tuned blocks-to-live
+// those are only gonna be enabled with DynamoRouteCachingProvider.newCachedRoutesRolloutPercent anyway
+export const DEFAULT_BLOCKS_TO_LIVE_ROUTES_DB: { [chain in ChainId]: number } = {
+  [ChainId.MAINNET]: 10,
+  [ChainId.GOERLI]: 10,
+  [ChainId.SEPOLIA]: 10,
+  // https://dune.com/queries/2009572
+  [ChainId.OPTIMISM]: 60,
+  [ChainId.OPTIMISM_GOERLI]: 60,
+  [ChainId.OPTIMISM_SEPOLIA]: 60,
+  [ChainId.ARBITRUM_ONE]: 60,
```

Comment:
arbitrum is a lot faster than optimism (about 8 times) which means that this would update a lot more often; we store multiple routes even per ratio right? shall we do it that they update it every 1 hour-ish? That way we can still leverage the same one many times, but also recaculate more often if the last one that we chose was more than lets say 2 minutes old;

Reply:
offline discussed, we decided to go with 10 minutes worth of blocks-to-live

Reply:
probably cache should expire based on time, blockNumber based expiration makes it more complex.
but that's to keep in mind for router next, not worth refactoring routing api now